api/cron/DONT_TOUCH_ENDPOINTS.txt

```diff
@@ -0,0 +1 @@
+qagen-r3-1,qagen-r3-2,qagen-r3-3
```

Comment:
Maybe we should make every endpoint name have it's own line. Might be easier to visualize if the list gets long.

Reply:
Actually, douwe and I think that it also makes sense to not commit the actual exclude list.